CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+# Changelog
```

Comment:
Do we need to change anything in this doc?

https://github.com/envoyproxy/nighthawk/blob/main/source/server/README.md#response-options-config

Could you also check if there are any other README docs or similar that may need updates due to these changes?

Reply:
Done. There don't appear to be other docs to change. Did a search for the phrases ResponseOptions and response_options for anything that might reference the configuration, and found only this and a couple other files I had already fixed.